### PR TITLE
To do list 

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
This PR is raised to submit for code review in the Microvers dashboard of Mohammad Rafi Amin.

Description of the task to be completed 👎

Add the HTML for the structure.
Add the CSS file for the styling.
Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
use JS structures
💯
Use WebPack structure
Use modules for better organizing
Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.

### CODE REVIEW

**Comment by mrkamin** [comment]
submit for code review

File: src/modules/task-show.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  Array.from(clearToDoTasks).forEach((element) => {
+    element.addEventListener('click', (e) => {
+      e.stopPropagation();
+      /* || Delete from local storage and remove div */
+      const parentEl = element.parentElement;
+      deleteTask(parentEl.id);
+      parentEl.remove();
+      counter -= 1;
+    });
+  });
```
Comment (Gopxfs) [inline src/modules/task-show.js]: Hello, @mrkamin your project is looking and working great 😄!
As requested on the project requirements:
image

Your tasks should have their indexes updated after being deleted. Currently, the lack of this feature is causing a bug 😉:
If I add tasks with the same name:
image

And delete one of them:
image
Please notice that the indexes weren't updated.

Then, if I try to add a new one with the same name, it doesn't work:
image

I know this sounds like something that may never happen with your actual code, but as programmers, we need to make sure our code works as intended for every possible scenario 😄. Please make sure to update your indexes after removing a task 😉. Cheers!

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @mrkamin 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Nice work with your design 😉.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/task-show.js (hunk 1 above)
Comment (mrkamin) [reply src/modules/task-show.js]: @Gopxfs,
thanks for your good and valuable feedback.
I will take in to account your comments.

**Review by mrkamin — COMMENTED** [review]
Hi @mrkamin 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Excellent communication with the reviewers

Cheers and Happy coding!👏👏👏

Feel free to mention @mrkamin to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**